lib/child_process.js

```diff
@@ -376,6 +376,8 @@ function execFile(file /* , args, options, callback */) {
   }
   if (options.signal) {
     if (options.signal.aborted) {
+      if (!ex)
```

Comment:
Is there any condition where `ex` will be already defined? Or should the check be removed?

Reply:
Nope, there's no way to get there with a value other than `null.` I'm going to extract the whole exit-with-AbortController to its own function, though.